test/serializers/adapter_for_test.rb

```diff
@@ -20,7 +22,7 @@ def test_overwrite_adapter_with_symbol
         adapter = ActiveModel::Serializer.adapter
         assert_equal ActiveModel::Serializer::Adapter::Null, adapter
       ensure
-
+        ActiveModel::Serializer.config.adapter = @previous_adapter
```

Comment:
You'll notice it was an empty ensure before I got to it....  It could go in an after each teardown... This was just a minimal fix